bears/general/TextLintBear.py

```diff
@@ -0,0 +1,194 @@
+            This rule checks whether your sentence starts with a duplicated
+            conjunction.
+        :param no_empty_section:
+            This rule does not allow to create an empty section.
+            For example, there is an empty section ``# Header A`` below:
+            ```
+            # Header A
+
+            # Header B
+
+            Text.
+            ```
+        :param check_date_weekday_mismatch:
+            This rule finds a mismatch between a date and the corresponding
+            weekday.
+        :param check_grammar:
+            This rule checks your English grammar with Ginger Proofreading.
+        :param max_lines_per_file:
+            Number of lines allowed per file.
+        :param max_comma_per_sentence:
+            Number of commas allowed per sentence.
+        :param check_no_good_words:
+            Set of NG (No Good) words to check for.
+        :param period_in_list_item:
+            Checks whether a sentence in a list item ends with a period.
+        :param minimum_acronym_length:
+            Minimum length for unexpanded acronyms.
+        :param maximum_acronym_length:
+            Maximum length for unexpanded acronyms.
+        :param ignore_acronyms:
+            A list that contains the acronyms to ignore.
+        :param check_rousseau:
+            This rule checks English sentence using rousseau, which is a
+            lightweight proofreader written in Javascript.
+            It can check:
+            - Passive voice
+            - Lexical illusions – cases where a word is repeated
+            - 'So' at the beginning of the sentence
+            - Adverbs that can weaken meaning: really, very, extremely, etc.
+            - Readibility of sentences
+            - Simpler expressions
+            - Weasel words
+            - If a sentence is preceded by a space
+            - If there is no space between a sentence and its ending
+              punctuation
+            - If sentences are starting with uppercase letter
+        :param check_offensive_expressions:
+            This rule helps you find gender favouring, polarising, race
+            related, religion inconsiderate, or other unequal phrasing.
+        :param check_common_misspellings:
+            This rule helps to find common misspellings from Wikipedia's
+            list of common misspellings.
+        :param write_good:
+            This rule checks your English styles with `write-good
+            <https://github.com/btford/write-good>`_.
+        :param check_relative_links:
+            This rule enables dead link checks against relative URIs.
+            Note that you also have to specify the ``base_uri`` to make this
+            option work.
+        :param base_uri:
+            The base URI to be used for resolving relative URIs.
+        :param ignore_links:
+            A list of URIs to be ignored, i.e. skipped from availability
+            checks.
```

Comment:
have you checked out `InvalidLinkBear` if there's a similar setting? If so we should call this one the same :+1: